src/Yaml.ts

```diff
@@ -17,6 +17,12 @@
 import * as yaml from "js-yaml";
 import * as _ from "lodash";
 
+// tslint:disable-next-line:interface-over-type-literal
+export type Options = {
```

Comment:
Any reason _not_ to use an interface here and avoid the tslint warning/disable?

Reply:
None against, none for it either.

Interfaces seem to mean much more than what this simple type definition should carry.